pages/plans/[id].js

```diff
@@ -27,6 +28,15 @@ const PlanSummary = ({ hackneyToken, plan }) => {
     setEditGoal(false);
   };
 
+  const getActionsList = () => {
+    if (_plan.goal) {
+      if (_plan.goal.actions) {
+        if (_plan.goal.actions.length > 0)
+          return <ActionsList actions={_plan.goal.actions} />;
+      }
+    }
+  };
```

Comment:
You can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to simplify this
```suggestion
  const getActionsList = () => {
          return <ActionsList actions={_plan?.goal?.actions || []} />;
  };
```

I don't think we need to worry about checking the length as `ActionsList` will return nothing for an empty array anyway?

Reply:
can probably just make it all inline {!editGoal && <ActionsList actions={_plan.goal?.actions || []} />}

Reply:
it returns the headers of the table for an empty array, so I don't know if that's what we want